firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/messages.txt

```diff
@@ -0,0 +1,207 @@
+/* Write function
+Scenario: Message: The write method correctly generates an encoded body from the protobuf message descriptor payload
+
+  GIVEN("A Message object default-constructed with StateSegment Taggedunion, default message types and a max size of 252 bytes as its class template parameters")
+  // assertions carried out in given:
+  // message class type member is 0
+
+    // setup:
+    // message class type member set to 5
+    // payload of parameters_request is set
+    // message descriptor array containing one element corresponding to Unknown message
+
+    WHEN("Write method is called on a message object with it's value of the payload tag greater than the size of the descriptor array")
+      THEN("The write method reports invalid type status")
+      THEN("After the write method is called, the type member remains unchanged")
```

Comment:
Now that we removed `auto type = ...`, the type member should be changed on every `write` method call, right? (This applies to all tests of the write method)